client/src/EditEntry.jsx

```diff
@@ -0,0 +1,63 @@
+import { useState } from 'react';
+import { data } from './data.js';
+
+export default function EditEntry() {
+  const [title, setTitle] = useState('');
+  const [photoURL, setPhotoURL] = useState('');
+  const [notes, setNotes] = useState('');
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    const formValues = {
+      title: title,
+      photoUrl: photoURL,
+      notes: notes,
+    };
+    console.log(data.entries);
+    if (data.editing === null) {
+      formValues.entryId = data.nextEntryId++;
+      data.entries.unshift(formValues);
+    } 
+    // else {
+    //   formValues.entryId = data.editing.entryId;
+    //   updateEntries(formValues);
+    //   const updatedLi = renderEntry(formValues);
+    //   const liToReplace = findLi(data.editing.entryId);
+    //   liToReplace.replaceWith(updatedLi);
+    //   updateFormTitle('New Entry');
+    //   data.editing = null;
+    // }
+    // resetForm();
+  }
+
+  return (
+    <div>
+      <h3>New Entry</h3>
+      <img
+        src={photoURL ? photoURL : '/images/placeholder-image-square.jpg'}
+        alt="placeholder"
+      />
+      <form onSubmit={handleSubmit}>
+        <label htmlFor="title">Title</label>
```

Comment:
`htmlFor` requires an `id` in the `input` element. I think it's best to wrap the label around the input instead of using `htmlFor`:
```
<label>
  Title
  <input name="title" ... />
</label>
```